webtool/frontend/src/components/cost-section.tsx

```diff
@@ -0,0 +1,80 @@
+import {FunctionComponent} from "react"
+import {DataList, Heading} from "@radix-ui/themes"
+import { PiMoneyLight } from "react-icons/pi"
+
+export const CostDisplay: FunctionComponent<{ artifact: any }> = ({artifact}) => {
```

Comment:
If you add argument `{displayCostPerKwh = true}` of type `{displayCostPerKwh?: boolean}` we can use it to hide it for batteries.

Or `hideCostPerKwh`, whichever you prefer.